roles/rabbitmq/tasks/main.yml

```diff
@@ -12,6 +12,14 @@
    enabled: yes
   when: manage_services|default(false)
 
+- name: Delete guest user on rabbitmq 
+  rabbitmq_user:
+     user:  '{{ rabbitmq_default_user }}' 
+     state: absent 
+  notify:
+      - restart rabbitmq
```

Comment:
Do we need to restart rabbitmq? IMHO server should take that into account even without restarting right?